src/Kip.scss

```diff
@@ -196,6 +196,7 @@
     top: 13px;
     width: calc(100% - 126px);
     left: 45px;
+    z-index: 2 !important;
```

Comment:
why does this fix the issue ?

Reply:
It place it above the other element, we can't access to the seekbar because some element hide it.

Reply:
what change in the player version caused this to start happening ?

Reply:
maybe this - https://github.com/kaltura/playkit-js-ui/commit/49557f0bd3a53e5b048b2e3533d9b4a4fcca1ae8

Reply:
that change [was reverted though](https://github.com/kaltura/playkit-js-ui/commit/5983a8702aa38618df367bb204e8720b3e7cebfb)

Reply:
caused from https://github.com/kaltura/playkit-js-ui/commit/35747d0881adeb865603d6e22ece614486b2fb2f#diff-97142631118c5fe6a3af6002847a92895570302e7edd7ee84ade0b553b6008f7L61, the css changes.

Reply:
basically z-index should be fine, since in IV player we don't have plugins that may be damaged by it. however, the issue that is causing it is not clear to me.. what element is overlaying the seekbar and preventing interaction?

Reply:
The left-controls and the right controls cover the seek bar. it's caused from this change https://github.com/kaltura/playkit-js-ui/commit/35747d0881adeb865603d6e22ece614486b2fb2f#diff-97142631118c5fe6a3af6002847a92895570302e7edd7ee84ade0b553b6008f7L61 - the flex adding and the float removing

Reply:
seek bar shouldn't cover the other controls. it's only because there was no css handling in case cc button is displayed on the player.
adding the css. now there is not overlapping and we can access to all buttons without any issue.

Reply:
even though `z-index: 2` will bring back the interaction with the seek bar, the entire bottom bar doesn't look good, specifically the right side:
![image](https://github.com/user-attachments/assets/e607a27f-7b72-4a6c-8bee-d219b5df21fd)
now some of the right controls are not accessible, because the seek bar is overlaying the right controls.
I suggest trying to fix it in another way- where all the bottom bar elements are accessible and no element is overlaying another element.

Reply:
I will approve, but I feel like we should make changes in the bottom bar for kiv - where the seekbar goes between the left and right controls, rather than using z-indexes; this is a band-aid and not a real solution. Please open a FEC ticket dedicated to at least check the possibility of these kind of changes. Thanks